.github/workflows/publish-brew-package.yml

```diff
@@ -56,9 +56,16 @@ jobs:
 
       - name: Download the distribution tar.gz file
         run: |
-          HZ_PACKAGE_URL=$(mvn --batch-mode dependency:copy -Dartifact=com.hazelcast:${HZ_DISTRIBUTION}-distribution:${HZ_VERSION}:tar.gz \
-          -DoutputDirectory=./ -Dmdep.useBaseVersion=true | grep 'Downloaded from' | grep -Eo "https://[^ >]+${HZ_DISTRIBUTION}-distribution-.*.tar.gz")
-          echo "HZ_PACKAGE_URL=$HZ_PACKAGE_URL" >> $GITHUB_ENV
+          . .github/workflows/build.functions.sh
+          export HZ_SNAPSHOT_INTERNAL_PASSWORD=${{ secrets.HZ_SNAPSHOT_INTERNAL_PASSWORD }}
+          export HZ_SNAPSHOT_INTERNAL_USERNAME=${{ secrets.HZ_SNAPSHOT_INTERNAL_USERNAME }}
```

Comment:
Why this vs setting an `env`?

Reply:
To keep it closer to the usage, to show where they're used. Also, When we decide to remove the step we'll remove these envs as well, without leaving global leftovers

Reply:
right, well,  just a force of habit. You want me to change it?

Reply:
> right, well, just a force of habit. You want me to change it?

I think that would be clearer?